src/components/PieChart.js

```diff
@@ -0,0 +1,161 @@
+
+import React from 'react';
+import Highcharts from 'highcharts';
+import drilldown from 'highcharts/modules/drilldown.js';
+import noDataToDisplay from 'highcharts/modules/no-data-to-display.js';
+
+function getCategoriesSeriesData(datas) {
+	datas = datas[0].data;
+	datas.sort(function(a, b){return b[1] - a[1]});
```

Comment:
This is hard to read, since I'm not sure what you're sorting by. What is b[1] and a[1]. You could add a comment here, or could expand the function and make it more explicit with naming.